contents/docs/runbook/services/plugin-server/ingestion-lag.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Ingestion lag
+sidebar: Docs
+showTitle: true
+---
+
+> **Important:** Please ensure you [have access to production](/handbook/engineering/how-to-access-posthog-cloud-infra) so that you are able to handle incidents!
+
+## Background
+
+Team Pipeline has committed to processing "standard" events end-to-end within 30 seconds (p95) and "buffered" events within 1m30s (p95). 
+
+It's important for customers that their data is queryable soon after being sent so they can debug their implementation and get the feeling that PostHog is working correctly.
+
+To calculate how we're doing with respect to our commitment, we have a Celery task that every 60 seconds calculates lag based on an event that we emit every minute from a plugin. 
+
+As a result of this, our metric includes 60 second error bars, meaning **we actually target 1m30s for "standard" events and 2m30s for "buffered" events**.
+
+Also, beyond measuring how fast we are, the ingestion lag metric being high might also be an **indication that we're not processing events at all**, so it's important that we treat it seriously.
+
+## Debugging
```

Comment:
Please incorporate info from https://github.com/PostHog/product-internal/blob/main/infrastructure/runbooks/alerts.md#e2e-ingestion-lag & make sure to update the link in pager duty later too: https://posthog.pagerduty.com/incidents/Q0O67KNGNTZ73A?utm_source=slack&utm_campaign=channel later